remote-api/src/main/resources/alfresco/templates/webscripts/org/alfresco/repository/site/membership/potentialmembers.get.js

```diff
@@ -3,7 +3,7 @@ function main()
    // Get the args
    var siteShortName = url.templateArgs.shortname,
       site = siteService.getSite(siteShortName),
-      filter = (args.filter != null) ? args.filter : (args.shortNameFilter != null) ? args.shortNameFilter : "",
+      filter = (args.filter != null) ? args.filter + " [hint:useCQ]" : (args.shortNameFilter != null) ? args.shortNameFilter : "",
```

Comment:
Wouldn't it be better to always add the hint regardless of the filter/shortNamefilter being populated?
E.g.: `((args.filter != null) ? args.filter : (args.shortNameFilter != null) ? args.shortNameFilter : "") + " [hint:useCQ]"`